hnn_core/network_builder.py

```diff
@@ -411,9 +392,10 @@ def _create_cells_and_feeds(self, threshold, record_vsoma=False,
                 else:
                     BasketCell = type2class[src_type]
                     cell = BasketCell(src_pos, gid=gid)
-                if _short_name(src_type) in self.net.feed_times['tonic']:
-                    cell.add_tonic_input(
-                        **self.net.feed_times['tonic'][_short_name(src_type)])
+                if ('tonic' in self.net.external_biases and
+                        src_type in self.net.external_biases['tonic']):
```

Comment:
Curious if there's any instance where src_type is not present? No need to change either way, just wondering if this is a real problem that would crop up.

Reply:
Oh yeah. That's actually the case most often than not: you are only likely to insert a current clamp onto a single cell type, say, `L5_pyramidal`, and leave the rest alone.